generator/iso-639-localized.csv

```diff
@@ -0,0 +1,9 @@
+ID;Localized Name
+nob;Norsk Bokmål
```

Comment:
Synes det burde hete `Norsk (bokmål)` kanskje. Stor `B` i `Bokmål` ser feil ut for meg.